com.valvesoftware.Steam.CompatibilityTool.Proton.yml

```diff
@@ -172,6 +173,45 @@ modules:
     sources: *wine_sources
 
 
+  - name: media-converter
+    subdir: media-converter
+    build-options:
+      append-path: /usr/lib/sdk/rust-stable/bin
+      env:
+        CARGO_HOME: /run/build/media-converter/media-converter/cargo
+        RUST_TAGET: x86_64-unknown-linux-gnu
```

Comment:
```suggestion
        RUST_TARGET: x86_64-unknown-linux-gnu
```

Reply:
Looks like VS Code helped me make the same typo in all occurrences, so it worked regardless. Thanks.